# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../argus-clients/drng-hotfix.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./include/argus_util.h.orig	2006-12-14 00:30:11.000000000 +0100
+++ ./include/argus_util.h	2006-12-14 00:30:38.000000000 +0100
@@ -595,7 +595,7 @@
 #define ARGUSPRINTSTARTRANGE		103
    { "srng", "", 6 , 1, ArgusPrintStartRange, ArgusPrintStartRangeLabel},
 #define ARGUSPRINTENDRANGE		104
-   { "erng", "", 6 , 1, ArgusPrintEndRange, ArgusPrintEndRangeLabel},
+   { "drng", "", 6 , 1, ArgusPrintEndRange, ArgusPrintEndRangeLabel},
 #define ARGUSPRINTTCPSRCBASE		105
    { "stcpb", "", 12 , 1, ArgusPrintTCPSrcBase, ArgusPrintTCPSrcBaseLabel},
 #define ARGUSPRINTTCPDSTBASE		106
--- ./support/Config/ra.print.all.conf.orig	2006-12-14 00:31:44.000000000 +0100
+++ ./support/Config/ra.print.all.conf	2006-12-14 00:31:17.000000000 +0100
@@ -27,4 +27,4 @@
 RA_FIELD_DELIMITER=','
 RA_USEC_PRECISION=6
 RA_PRINT_NAMES=port
-RA_FIELD_SPECIFIER= srcid stime ltime trans flgs dur avgdur stddev mindur maxdur saddr daddr proto sport dport stos dtos sdsb ddsb sttl dttl sipid dipid pkts spkts dpkts bytes sbytes dbytes appbytes sappbytes dappbytes sload dload load sloss dloss loss sploss dploss ploss srate drate rate smac dmac dir sintpkt dintpkt sintpktact dintpktact sintpktidl dintpktidl sintpktmax sintpktmin dintpktmax dintpktmin sintpktactmax sintpktactmin dintpktactmax dintpktactmin sintpktidlmax sintpktidlmin dintpktidlmax dintpktidlmin jit sjit djit jitact sjitact djitact jitidl sjitidl djitidl state ddur dstime dltime dspkts ddpkts dsbytes ddbytes pdspkts pddpkts pdsbytes pddbytes suser duser tcpext swin dwin jdelay ldelay seq bins binnum smpls dmpls svlan dvlan svid dvid svpri dvpri srng erng stcpb dtcpb tcprtt inode
+RA_FIELD_SPECIFIER= srcid stime ltime trans flgs dur avgdur stddev mindur maxdur saddr daddr proto sport dport stos dtos sdsb ddsb sttl dttl sipid dipid pkts spkts dpkts bytes sbytes dbytes appbytes sappbytes dappbytes sload dload load sloss dloss loss sploss dploss ploss srate drate rate smac dmac dir sintpkt dintpkt sintpktact dintpktact sintpktidl dintpktidl sintpktmax sintpktmin dintpktmax dintpktmin sintpktactmax sintpktactmin dintpktactmax dintpktactmin sintpktidlmax sintpktidlmin dintpktidlmax dintpktidlmin jit sjit djit jitact sjitact djitact jitidl sjitidl djitidl state ddur dstime dltime dspkts ddpkts dsbytes ddbytes pdspkts pddpkts pdsbytes pddbytes suser duser tcpext swin dwin jdelay ldelay seq bins binnum smpls dmpls svlan dvlan svid dvid svpri dvpri srng drng stcpb dtcpb tcprtt inode
